src/pages/student/page.tsx

```diff
@@ -1,11 +1,74 @@
-import { Layout } from "antd";
+import { Link } from "atomic-router-react";
+import {
+  Card,
+  Col,
+  Layout,
+  Row,
+  Space,
+  Typography
+} from "antd";
+import { useUnit } from "effector-react";
+import { SettingOutlined } from "@ant-design/icons";
 import { PageHeader } from "../../shared/ui/PageHeader/PageHeader";
+import { routes } from "../../shared/lib/atomic-router/route";
+import {
+  $user,
+  $userSettings
+} from "./model";
+import { homeworkCards } from "./settings/mocks";
+import styles from "./page.module.css";
 
 const { Content } = Layout;
+const { Text ,Title } = Typography;
 
-export const StudentPage = () => (
+export const StudentPage = () => {
+  const [
+    user,
+    userSettings
+  ] = useUnit([
+    $user,
+    $userSettings
+  ]);
+
+  return (
   <Layout>
     <PageHeader title="Личный кабинет" />
-    <Content />
+    <Content className={styles._}>
+      <Row>
+        <Space size={100}>
+          <Col span={50}>
```

Comment:
span max 24, maybe it's better to use one col for table and the second for user links?